apps/ecs/views/ineligible-tupe.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p class="govuk-body">
+      {{#t}}pages.ineligible-employee.ineligible-employee-reason-1{{/t}}
```

Comment:
this points to content:  '"If the employee has been working for you, without a break in employment, from a date before 29 February 2008, then you do not need to request a right to work check." 

The /ineligible-tupe page should have content:  "If the employee has been working for you as a result of a TUPE transfer, without a break in employment, from a date before 29 February 2008, then you do not need to request a right to work check. Document checks should have been carried out when the person was recruited.